.github/workflows/test.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.16.x, 1.17.x, 1.18.x]
+        go-version: [1.17.x, 1.18.x, 1.19.x]

```

Comment:
`unsafe.Slice` is available since go1.17. We don't use this backend on go1.16, so this change is harmless.